src/main/java/module_two_1/models/enums/FacultiesNames.java

```diff
@@ -0,0 +1,14 @@
+package module_two_1.models.enums;
+
+public enum FacultiesNames {
```

Comment:
It is advised to insert a blank line after class declaration

Reply:
Added blank line